backend/app/interactem/app/api/routes/pipelines.py

```diff
@@ -263,78 +253,57 @@ def delete_pipeline(
     return Message(message="Pipeline deleted successfully")
 
 
-@router.post("/{id}/revisions/{revision_id}/run", response_model=PipelineRevisionPublic)
-async def run_pipeline(
+@router.get("/{id}/runs", response_model=PipelineRunsPublic)
```

Comment:
Are these actually runs? Or is deployment or instantiation more appropriate? They are pipelines that have been deployed to an agent? As in there might not be data running through the pipeline?